package.json

```diff
@@ -1,4 +1,94 @@
 {
   "name": "builder-react-component",
-  "version": "0.0.0"
+  "version": "0.0.1",
+  "description": "Builder Archetype - React Component",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/FormidableLabs/builder-react-component.git"
+  },
+  "author": "Ryan Roemer <ryan.roemer@formidablelabs.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/FormidableLabs/builder-react-component/issues"
+  },
+  "homepage": "https://github.com/FormidableLabs/builder-react-component",
+  "scripts": {
+    "npm:postinstall": "node -e \"require('fs').stat('lib', function(e,s){process.exit(e || !s.isDirectory() ? 1 : 0)})\" || builder run build-lib",
+    "npm:preversion": "builder run check",
+    "npm:version": "builder run clean && builder run build && git add -A dist",
+    "npm:test": "builder run test-frontend",
+    "clean-dist": "rimraf dist",
+    "build-dist-min": "webpack --config node_modules/builder-react-component/config/webpack/webpack.config.js",
```

Comment:
@boygirl the other side of that coin is that it's harder to know how much you've diverged from the "off the shelf" behavior and *why*. i.e. did I add this command *just* to change what file it's pointing at? Or are the flags different too? Did I explicitly add/remove any of the flags that are different, or did my archetype just get updates? etc.